config/default.js

```diff
@@ -9,25 +9,24 @@ const contactAddress = 'hello@inventaire.io'
 const config = module.exports = {
   name: 'inventaire',
   env: 'default',
-  host: 'localhost',
   universalPath: require('./universal_path'),
   // Only http is supported: in production, TLS is delegated to Nginx
   // see http://github.com/inventaire/inventaire-deploy
   // protocol: 'http'
   verbose: true,
+  hostname: 'localhost',
   protocol: 'http',
   port,
   // Override in ./local.js when working offline to prevent trying to fetch remote resources (like images) when possible
   offline: false,
-  fullHost: function () {
-    return `${this.protocol}://${this.host}:${this.port}`
+  getLocalOrigin: function () {
+    return `${this.protocol}://${this.hostname}:${this.port}`
   },
   publicProtocol: 'http',
-  publicHost: 'localhost',
-  fullPublicHost: function () {
-    return `${this.publicProtocol}://${this.publicHost}:${this.port}`
+  publicHostname: 'localhost',
```

Comment:
`publicHostname` & `hostname` have the same key value

Reply:
by default yes, but not in all environment: typically, our `config/local-production.js` file would contain:
```js
{
  publicHostname: 'inventaire.io'
}
```
while `hostname` would still use the default value : `localhost`. Should `hostname` be renamed `localHostname` or `privateHostname` to make the distinction clearer?

Reply:
i was more thinking of 
`publicHostname: config.hostname`
and environments who need a specific `publicHostname` key could define it in their config file

Reply:
ok